motion_library_layer/motion_sideward/src/sidewardTest.cpp

```diff
@@ -79,6 +99,10 @@ int main(int argc, char **argv)
   ros::NodeHandle nh;
   // Subscribing to feedback from ActionServer
   ros::Subscriber sub_ = nh.subscribe<motion_commons::SidewardActionFeedback>("/sideward/feedback", 1000, &sidewardCb);
+  ros::Subscriber ip_data_sub = nh.subscribe<std_msgs::Float64MultiArray>("/varun/sensors/front_camera/ip_data", 1000,
```

Comment:
i'll need to change it in some other files to so i'm merging it right now and then i'll make an another pull request that will change this topic name in all the nodes